### PR TITLE
LCMS: 3 changes--um to micron; A to a ; mass range unit m/z

### PR DESCRIPTION
**Pull request by cebriggs7135**
Added mass_range_unit to m/z.
Changed lc_id_unit enum: from um to micron
Changed the following fieldnames:
    lc_mobile_phase_A to lc_mobile_phase_a
    lc_mobile_phase_B to lc_mobile_phase_b

### CODE REVIEW

File: src/ingest_validation_tools/table-schemas/level-2/lcms.yaml [hunk 1]
```diff
@@ -27,6 +27,11 @@ fields:
   -
     name: mass_range_high_value
     description: The high value of the scanned mass range for MS1. (unitless)
+  -
+    name: mass_range_unit
+    description: Units of the mass range values
+      enun
+       - m/z
```
Comment (mccalluc) [inline src/ingest_validation_tools/table-schemas/level-2/lcms.yaml:34]: - mass range is literally unitless.

File: src/ingest_validation_tools/table-schemas/level-2/lcms.yaml [hunk 2]
```diff
@@ -80,7 +85,7 @@ fields:
     description: units of LC column inner diameter (typically microns)
     constraints:
       enum:
-        - um
+        - micron
```
Comment (mccalluc) [inline src/ingest_validation_tools/table-schemas/level-2/lcms.yaml:88]: Unless one community strongly prefers micron to um, I think we should be consistent?

File: src/ingest_validation_tools/table-schemas/level-2/lcms.yaml [hunk 3]
```diff
@@ -97,10 +102,10 @@ fields:
     name: lc_gradient
     description: LC gradient
   -
-    name: lc_mobile_phase_A
+    name: lc_mobile_phase_a
     description: Composition of mobile phase A
   -
-    name: lc_mobile_phase_B
+    name: lc_mobile_phase_b
```
Comment (mccalluc) [inline src/ingest_validation_tools/table-schemas/level-2/lcms.yaml:108]: I'll make a separate PR with these.

File: src/ingest_validation_tools/table-schemas/level-2/lcms.yaml (hunk 1 above)
Comment (cebriggs7135) [reply src/ingest_validation_tools/table-schemas/level-2/lcms.yaml:34]: Yes. It is a ratio of the mass value divided by the charge value. m/z describes how the mass is calculated. mass spectrometry output is called "m/z", not just "mass". z is usually 1 but not always.

File: src/ingest_validation_tools/table-schemas/level-2/lcms.yaml (hunk 2 above)
Comment (cebriggs7135) [reply src/ingest_validation_tools/table-schemas/level-2/lcms.yaml:88]: "micron" is the standard, I am discovering. If you keep, then I will have to change & resubmit a number of metadata.tsvs.

**Pull request closed without merging**